test/e2e/util_test.go

```diff
@@ -850,7 +850,6 @@ func getRouteHost(t *testing.T, route *routev1.Route, router string) string {
 		}
 	}
 
-	t.Fatalf("failed to find host name for default router in route: %#v", route)
```

Comment:
Any reason not to drop the other call to fatal in this function? Would that perturb other test behaviour in a bad way? If the other call can be dropped we can drop the call to t.Helper() too.

Reply:
I don't think it would perturb the test behavior, removed.